### ISSUE TITLE
MAX - Wrong default audio 

### ISSUE THREAD

**Issue by matthuisman** [matthuisman/slyguy.addons#926]
@GG-76
"I found out that I don't have a problem with subtitles, I have a problem with the sound in Max. Suddenly, Czech is constantly being selected as my default sound language, even though I haven't changed the settings. In the default sound settings in Max, I have this: "default,original,interface,en" so I don't know if there's a problem with that. For example, in the Dune Prophecy series, the Czech stream is displayed as the default in the streams, but I don't know what affects it and how I can change it. In Disney+ works that right."

**Comment by matthuisman** [comment]
@GG-76

I tried to replicate the above - but mine get set to default English.

Please advise what your Kod interface language is set to?
Is it Czech?

**Comment by matthuisman** [comment]
ive just changed the default_audio to be default,original,en
So itll ignore interface language

**Comment by GG-76** [comment]
@matthuisman yes my interface is Czech. I tried removing the interface in the settings, like you did, and it works correctly now. thank you.

**Comment by matthuisman** [comment]
I think for audio, most would prefer it to be the original audio and not
dubbed. So I changed the default settings to reflect that.

Subs still need some work to get the best behavior

**Comment by matthuisman** [comment]
@TobiasvonCarnap  this issue is for MAX addon.
Think you've posted in the wrong location?

**Comment by matthuisman** [comment]
Please open a new issue if needed :)
